lib/pymedphys/_experimental/autosegmentation/pipeline.py

```diff
@@ -19,7 +19,7 @@
 
 from pymedphys._imports import numpy as np
 from pymedphys._imports import pydicom
-from pymedphys._imports import tensorflow as tf
+from pymedphys._imports.slow import tensorflow as tf
```

Comment:
Throughout the code base, to import tensorflow, it now needs to be imported from a separate module. It is no longer included within `pymedphys._imports` instead, it is only included within `pymedphys._imports.slow`